Substitute about:debugging#/runtime/this-firefox for about:debugging (#488)
mdn/webextensions-examples: native-messaging/README.md

## Testing the example ##

> Then just install the add-on as usual, by visiting `about:debugging#/runtime/this-firefox` or click on the button to debug "This Firefox" , clicking "Load Temporary Add-on", and selecting the add-on's "manifest.json".

Reviewer: In nightly the menu option is "This Nightly"
```suggestion
First, install the add-on. Visit `about:debugging#/runtime/this-firefox` or, from `about:debugging` click "This Firefox" (or "This Nightly" in the Nightly version of Firefox), click "Load Temporary Add-on", and open the add-on's "manifest.json".
```
